make/autoconf/flags-cflags.m4

```diff
@@ -134,6 +134,9 @@ AC_DEFUN([FLAGS_SETUP_WARNINGS],
 
       WARNINGS_ENABLE_ALL="-W3"
       DISABLED_WARNINGS="4800"
+      if test "x$TOOLCHAIN_VERSION" = x2017; then
+          DISABLED_WARNINGS+=" 4307"
```

Comment:
Seems like this file uses 2 spaces as indenting, 4 spaces are used here.

Reply:
You Are right, I adjusted the indenting to 2 sapces and did a git commit  --amend and repushed it.